tests/DPDK_Flow_Dump_with_LNT.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (c) 2022 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+DPDK Flow Dump with LNT
+"""
+
+# in-built module imports
+import time
+import sys
+from itertools import dropwhile
+from datetime import datetime, timedelta
+
+# Unittest related imports
+import unittest
+
+# ptf related imports
+from ptf.base_tests import BaseTest
+from ptf.testutils import *
+
+# framework related imports
+import common.utils.log as log
+import common.utils.p4rtctl_utils as p4rt_ctl
+import common.utils.test_utils as test_utils
+import common.utils.ovs_utils as ovs_utils
+import common.utils.gnmi_ctl_utils as gnmi_ctl_utils
+from common.utils.config_file_utils import (
+    get_config_dict,
+    get_gnmi_params_simple,
+    get_interface_ipv4_dict,
+    get_gnmi_params_hotplug,
+)
+
+class DPDK_Flow_Dump_with_LNT(BaseTest):
+    def setUp(self):
+        BaseTest.setUp(self)
+        self.result = unittest.TestResult()
+        test_params = test_params_get()
+        config_json = test_params["config_json"]
+
+        try:
+            self.vm_cred = test_params["vm_cred"]
+        except KeyError:
+            self.vm_cred = ""
+
+        self.config_data = get_config_dict(
+            config_json,
+            pci_bdf=test_params["lnt_pci_bdf"],
+        )
+        
+        self.gnmictl_params = get_gnmi_params_simple(self.config_data)
+        self.gnmictl_hotplug_params = get_gnmi_params_hotplug(self.config_data)
+        self.tap_port_list = gnmi_ctl_utils.get_tap_port_list(self.config_data)
+        self.link_port_list = gnmi_ctl_utils.get_link_port_list(self.config_data)
+        log.info(self.gnmictl_hotplug_params)
```

Comment:
this can be removed

Reply:
Hotplug need to be removed. but link_port and tap_list needed for vxlan configuration